docs/CHANGELOG STYLE GUIDE.md

```diff
@@ -0,0 +1,145 @@
+# CHANGELOG STYLE GUIDE
+
+*This style guide is partly inspired from [common-changelog](https://github.com/vweevers/common-changelog)*
+
+First, these entries are meant for our consumers from all levels so we need to keep them accessible enough for them. To make it easy on our side to achieve this, we’ve created this style guide for creating a new Changelog on a new version release.
+## General Format
+
+### 1. Version Header:
+- Format: `## [version-number]`
+- Example: `## [0.11.0]`
+  This header indicate which version the log entry description concern and follows [Semantic Versioning](https://semver.org/), it’s also sorted from the newest to latest semantic version.
+### 2. Date:
+- Format: `_YYYY-MM-DD_`
+- Example: `_2024-06-24_`
+  The date help to quickly get which version was available when and get the release timeline.
+### 3. Section Headers:
+Sections represent a group of changes scoped to an artefact or section.
+For now we restrict them to the following one to have consistency and avoid ambiguity:
+-  `### Spark`
+-  `### Catalog App`
+-  `### Icons`
+-  `### CI`
+### 4. Subsections:
+- Use subsections for specific features or changes within a component.
+- Example: `#### 🆕 New Features`
+
+### 5. Change Types:
+Sort content by importance.
+Skip content that isn't important.
+Link each change to further information.
+
+> [!NOTE]
+> You can inspire your self from [gitmoji ](https://gitmoji.dev/)
+
+Use emojis and clear keywords to categorize changes:
+- 🆕 New Feature: `🆕`
+- 🎨 Improvement: `🎨`
+- 💄 UI Change: `💄`
+- 🐛 Bug Fix: `🐛`
+- 🔧 Maintenance: `🔧`
+- 🚀 Performance Improvement: `🚀`
+- 📝 Documentation: `📝`
+- ⬆️ Dependency Update: `⬆️`
+- 🗑️ Deprecation/Removal: `🗑️`
```

Comment:
IMO the emoji on the left are enough, you don't need to re-add it at the end.
```suggestion
Use emojis and clear keywords to categorize changes:
- 🆕 New Feature
- 🎨 Improvement
- 💄 UI Change
- 🐛 Bug Fix
- 🔧 Maintenance
- 🚀 Performance Improvement
- 📝 Documentation
- ⬆️ Dependency Update
- 🗑️ Deprecation/Removal
```

Reply:
Done in 6730295